steps/approved-premises/applications/location-factors.ts

```diff
@@ -1,5 +1,5 @@
 import { type Page, expect } from '@playwright/test'
-import {selectOption} from "../../delius/utils/inputs";
+import { selectOption } from '../../delius/utils/inputs'
```

Comment:
Do we need the `.js` suffix on this import?  I've been meaning to look into why we do, as IntelliJ doesn't add them...

```suggestion
import { selectOption } from '../../delius/utils/inputs.js'
```